src/modules/CurrencyModule.h

```diff
@@ -110,6 +115,9 @@ class CurrencyModule : public Module
 		int bribeMaxAmount { 150 };
 		int bribeLeastAmount { 20 };
 		int bribeSuccessChance { 68 };
+
+		int lowRiskRewardStealSuccessChance{ 50 };
+		int highRiskRewardStealSuccessChance{ 30 };
```

Comment:
I think they should be in the `CurrencyModule.cpp`. No other events are using them and they only effect the CurrencySystem. I removed the defines to the `EventModule.h` because they are needed by both of the modules. But in this case I don't think its needed to move them. 🙂